Dockerfile

```diff
@@ -1,127 +1,26 @@
-### Stage 0 - Base image
-FROM node:10.15.3-alpine as base
-WORKDIR /app
-RUN \
-  mkdir -p dist node_modules \
-  && chown -R node:node . \
-  && apk --no-cache  update \
-  && apk --no-cache  upgrade \
-  && npm config set unsafe-perm true \
+# Build for developers through Docker:
```

Comment:
I don't see the interest for this Dockerfile. 

And as it remains at the root of the project  it is also confusing : we could think that it is the current "production ready" Dockerfile for blip (whereas it is in server/Dockerfile).